test/parallel/test-resources-usage.js

```diff
@@ -0,0 +1,12 @@
+// Copyright Joyent, Inc. and other Node contributors.
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const r = process.resourcesUsage();
+assert.ok(typeof r.minflt === 'number');
```

Comment:
It might be useful to also assert that the values are `>= 0`.

Reply:
As said on the PR, i believe it will render the test flaky. Since the test is run without doing anything else, there are generally no memory page swap and almost no IO request, but it can change from platform to platform i suppose.

Reply:
@vmarchaud These values are all unsigned when coming from libuv – so they have to be `>= 0`.

Reply:
My bad i read `> 0` on @cjihrig message, will correct !